lib/src/clojure_lsp/handlers.clj

```diff
@@ -40,7 +40,7 @@
      (loop [backoff# 1]
        (if (> (quot (- (System/nanoTime) ~'_time) 1000000) 60000) ; one minute timeout
          (log/warn "Timeout waiting for changes for body")
-         (if (:processing-changes @db/db)
+         (if (seq (:processing-changes @db/db))
```

Comment:
I have a commit locally where this is `(contains? (:processing-changes @db/db) ~uri)`, i.e., the handlers wait for changes to the file they are updating, instead of waiting for changes to any file. I'm not sure though... it'd be a very minor speed up because it's rare that there are changes to several files simultaneously. WDYT?

Reply:
sounds good to me, the only issue would be if analysis from a file would affect another file, but I don't see that an issue for now.